### DISCUSSION TITLE
load js file from public folder

### DISCUSSION THREAD

**Discussion by valjusha** [vercel/next.js#47740] [Help] [answered]
### Summary

Hello!
The question is, how can I import js files from a public folder?

My actions:

1) using the webpack plugin I copy the files I need
_(when I start the project, I see the files I need in the folder .next/public/worky)_
```
config.plugins.push(new CopyPlugin({
     patterns: [
           {
               from: "./node_modules/worky/src",
               to: "./public/worky",
           },
      ],
}))
```

2) I try to load the script using `<Script />` from (next/script)

3) always get 404 file

### Additional information

```js
"next": "^13.1.6",
```

**Comment by icyJoseph** [comment]
How are you loading the script? Could you share some code?

**Comment by valjusha** [comment, marked as answer]
I had an error, I copied the scripts to the wrong folder. you need not in the `.next` folder
